kahuna/public/js/edits/image-editor.html

```diff
@@ -75,6 +75,14 @@
             ></ui-archiver>
         </div>
 
+        <div class="result-editor__delete-image">
+            <gr-image-delete image="ctrl.image"
```

Comment:
I think we always use `-` in element names for browser compat/styling/etc reasons?